website/docs/api/kb.mdx

```diff
@@ -190,25 +190,27 @@ Restore the state of the knowledge base from a given directory. Note that the
 | `exclude`   | List of components to exclude. ~~Iterable[str]~~                                                |
 | **RETURNS** | The modified `KnowledgeBase` object. ~~KnowledgeBase~~                                          |
 
-## Candidate {id="candidate",tag="class"}
+## InMemoryCandidate {id="candidate",tag="class"}
 
-A `Candidate` object refers to a textual mention (alias) that may or may not be
-resolved to a specific entity from a `KnowledgeBase`. This will be used as input
-for the entity linking algorithm which will disambiguate the various candidates
-to the correct one. Each candidate `(alias, entity)` pair is assigned to a
-certain prior probability.
+A `InMemoryCandidate` object refers to a textual mention (alias) that may or may
```

Comment:
```suggestion
An `InMemoryCandidate` object refers to a textual mention (alias) that may or may
```

Reply:
PyCharm refactoring doesn't take articles into account :stuck_out_tongue:

Reply:
I'd drop "(alias)" though and phrase everything as "mention". I think "alias" is confusing, as mentions don't have to be aliases of entity names already (and even less so once we primarily work in the embedding space). What do you think?

Reply:
agreed!

Reply:
This wasn't addressed I think?

Reply:
Updated in https://github.com/explosion/spaCy/pull/12341/commits/1c937db3af6a66fa36fbb0538dc975450254d301.